src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -3,17 +3,36 @@
 import core.basesyntax.Storage;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    private static final int MAX_STORAGE_SIZE = 10;
+    private K[] keys = (K[]) new Object[MAX_STORAGE_SIZE];
+    private V[] values = (V[]) new Object[MAX_STORAGE_SIZE];
+    private int size = 0;
```

Comment:
it's zero by default
```suggestion
    private int size;
```